docs/api/internal/nu-designer-openapi.yaml

```diff
@@ -0,0 +1,361 @@
+openapi: 3.1.0
+info:
+  title: Nussknacker Designer API
+  version: ''
+paths:
+  /api/app/healthCheck:
+    get:
+      tags:
+      - App
+      summary: Application health check service
+      operationId: getApiAppHealthcheck
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HealthCheckProcessSuccessResponseDto'
+              examples:
+                Example:
+                  summary: Application is healthy
+                  value:
+                    status: OK
+  /api/app/buildInfo:
+    get:
+      tags:
+      - App
+      summary: Application info service
+      operationId: getApiAppBuildinfo
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BuildInfoDto'
+              examples:
+                Example:
+                  summary: Application build info response
+                  value:
+                    name: nussknacker
+                    version: 1.12.0-SNAPSHOT
```

Comment:
maybe we should return some syntetic version so user won't think that he/she works on some old snapshot. e.g. `1.234.0` or similar